### DISCUSSION TITLE
translations/Chinese

### DISCUSSION THREAD

**Discussion by Cuchannel** [ravignir/RekMOD#85] [General]
![Uploading Screenshot_20221114_152111_com.unciv.app.jpg…]()

**Comment by AutumnPizazz** [comment]
This should be created as a new branch and a merge request should be submitted, instead of posting a strange image file here.
However, I helped you upload the Properties in Simplified Chinese.

**Reply by SeventhM** [reply]
Huh... How long has discussions on github been a thing? This is the first time I've seen this
